packages/cli/src/CliUtils.ts

```diff
@@ -18,7 +18,7 @@ import { awsStorage } from "@metaplex-foundation/js-plugin-aws";
 import { S3StorageManager } from "./config/index.js";
 
 export class Constants {
-  static CLI_VERSION = "0.6.1";
+  static CLI_VERSION = "0.7.0";
```

Comment:
A bump in the minor version number makes this a forced update. Are any of the changes released since 0.6.1 worth forcing all consumers to update before submitting new versions?

Reply:
We mede 2 changes
1. Review time for new and updates to app.
2. Block debug apk in submission. 

Since this would improve the developer feedback and turnaround time, I think its worth to force update.

Reply:
The review SLAs probably aren't, but blocking debug APKs is a useful functional improvement